### ISSUE TITLE
Bug: Preview image missing of first item in subscribtions feed

### ISSUE THREAD

**Issue by apfelsprudel** [azzy9/plugin.video.rumble#25]
In the subscriptions feed is the first item preview image missing, sometimes the second item too.
Version: 1.5.2

Want to say 'thank you' for maintining this plugin. It is already really usefull and I use it every day.
I also have the github browser installed to be up to date.

**Comment by azzy9** [comment]
Hi,
There would only be 3 reasons why images are not shown,
1. The REGEX has failed to pick it up, but this would most probably cause the video not to show up at all
2. The image is in a format that Kodi doesn't like e.g. Webp
3. There is no Image to retrieve, this can be caused by content creators using a method to sync their Youtube videos to other platforms such as Odysee and Rumble. Which it first uploads the video then takes its time providing the thumbnails. Or it could be caused by the video platform taking its time to process the video and generate the Thumbnails if it is done automatically.

If it is the last 2 options there is nothing I can do about it. Since I cannot reproduce the issue I cannot really provide a fix unless I discover what the issue is.

Does this happen to these videos when you login onto the rumble site?
Does it happen to certain channels?

**Comment by apfelsprudel** [comment]
3. and 2. imho can not be the case.
It happens only when the video item is the first item on top, when a new video comes to the subscriptions feed and it drops to the second place, the preview image appears. I have found that it happens to all channels I am subscribed, but I am not sure. I am subscribed to many german channels, maybe cohesive to [https://github.com/azzy9/plugin.video.rumble/issues/24]
It also does not happen in the channel feed.

I have installed Kodi 20.2 on a Fire TV 4K.
Anyone else having the problem?
